src/pyramid/config/__init__.py

```diff
@@ -139,13 +139,17 @@ class Configurator(
     :term:`dotted Python name` to the same.  If it is ``None``, a default
     root factory will be used.
 
+    If ``security_policy`` is passed, it should be an instance of a
+    :term:`security policy` or a :term:`dotted Python name` to the same.
+
     If ``authentication_policy`` is passed, it should be an instance
     of an :term:`authentication policy` or a :term:`dotted Python
-    name` to the same.
+    name` to the same.  (Deprecated as of Pyramid 2.0 in favor of
+    ``security_policy``.)
 
     If ``authorization_policy`` is passed, it should be an instance of
     an :term:`authorization policy` or a :term:`dotted Python name` to
-    the same.
+    the same.  (Deprecated as of Pyramid 2.0 in favor of ``security_policy``.)
```

Comment:
versionchanged directives should be added to the bottom of the docstring noting the deprecations and the additions.

Reply:
@mmerickel do you mean `versionchanged` or `deprecated` directives? `glossary.rst` uses `deprecated`.

Reply:
I think if I were doing it myself I would've used versionchanged in cases where a feature (like a particular argument) is deprecated in a function but not we didn't deprecate the whole function. People may disagree with me though, that's just my opinion. Thoughts?

Reply:
I'd follow the intent of the directives as explained in their documentation, [versionchanged](http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionchanged) and [deprecated](http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated). IOW, the feature has greater weight than the code (function, method, arguments) when deciding which directive is most appropriate.

In this case, one feature is deprecated in favor of another.